src/microcosm.js

```diff
@@ -487,6 +423,80 @@ class Microcosm extends Emitter implements Domain {
   parallel(actions: Action[]) {
     return this.append('GROUP').link(actions)
   }
+
+  /* Private ------------------------------------------------------ */
```

Comment:
@efatsi I'm going to start moving private methods into a section like this. If we append an underscore to a method name, it'll get minified in the production pass. These tokens are shared across all of the modules, so if we _did_ need to access a private method in something like `Presenter`, it would be safe.